knowledge/pkg/datastore/documentloader/converter/soffice.go

```diff
@@ -53,20 +53,35 @@ func (c *SofficeConverter) Convert(ctx context.Context, reader io.Reader, source
 	}
 	_ = tempfile.Close()
 
+	logger := log.FromCtx(ctx)
+
+	profileDir, err := os.MkdirTemp(os.TempDir(), "libreoffice-profile-*")
+	if err != nil {
+		logger.Error("Failed to create soffice profile directory", "path", profileDir, "error", err)
+		return nil, fmt.Errorf("failed to create soffice profile directory: %w", err)
+	}
+	defer os.RemoveAll(profileDir)
+
 	// Convert the file using soffice
-	cmd := exec.Command("soffice", "--headless", "--convert-to", outputFormat, "--outdir", os.TempDir(), p)
+	cmd := exec.Command(
+		"soffice",
+		"--headless",
+		fmt.Sprintf("-env:UserInstallation=file://%s", profileDir),
```

Comment:
Just wanted to leave a comment here to make sure that the single dash (`-env`) was intentional.